lighthouse-cli/test/smokehouse/test-definitions/forms/form-expectations.js

```diff
@@ -284,6 +284,96 @@ const expectations = [
       requestedUrl: 'http://localhost:10200/form.html',
       finalUrl: 'http://localhost:10200/form.html',
       audits: {
+        'autocomplete': {
+          id: 'autocomplete',
```

Comment:
remove all of this except:

`score`
`details.items`

We don't do full snapshots of audit expectations in smoke tests, just of the results.